libs/components/src/next/form/checkbox/stories/vl-checkbox.stories-doc.mdx

```diff
@@ -22,6 +22,21 @@ import { VlCheckboxComponent } from '@domg-wc/components';
 
 <ArgTypes of={VlCheckboxStories.CheckboxDefault} />
 
+### Met value
+
+Indien er geen value meegegeven wordt aan de checkbox, wordt er tijdens het submitten van een form de value `on` teruggegeven.<br/>
+Dit is zoals de HTML5 standaard het voorschrijft. Indien er wel een value meegegeven wordt, zal deze value teruggegeven worden.
+
+<Canvas of={VlCheckboxStories.CheckboxValue} />
+
+## Read only
+
+Aangezien readonly er voor zorgt dat de value van een input niet aangepast kan worden, heeft readonly plaatsen op een checkbox geen effect aangezien het aan- en uitvinken de checked state aanpast en niet de value.<br/>
+Als je een checkbox als readonly wilt gebruiken, moet je het `disabled` attribuut meegeven en gebruik maken van een hidden input zodat de value toch mee met het form gesubmit wordt.<br/>
+Zie dat de value van de hidden input overeenkomt met de checked state en de value van de disabled checkbox.
+
```

Comment:
als ik het goed begrijp is dit dezelfde issue als ze gwn native input checkbox zouden gebruiken; lijkt me niet per sé zo'n courante use-case? maar laat gerust staan hoor - als ze dit dan willen implementeren is er meteen een voorbeeld

Reply:
Dat is idd hoe je het native ook zou oplossen, heb deze info er bijgezet omdat het afwijkt van hoe de readonly werkt bij de andere componenten.

Reply:
ok is goed 👍🏼